### ISSUE TITLE
How to use mlp for regression

### ISSUE THREAD

**Issue by tengshaofeng** [fchollet/keras#2399]
dear every one:
      all the mlp examples are classifying problem , how about the regression? can you tell me what i 
should do?

**Comment by keunwoochoi** [comment]
* Do not use `Softmax` activation at the output layer
* You might want to use `Sigmoid` if the output range is known and it's finite. In that case, the output range should be mapped into [0, 1]. It usually helps optimisation.
* Use loss functions such as `binary_crossentropy` or `mse` instead of `categorical_crossentropy`. `binary_crossentropy` is known to be better - as far as I know.

**Comment by tengshaofeng** [comment]
thanks for your help. Does binary_crossentropy  also fit for regresion problem? If I use sigmoid , but the output is range in  [0,1000], then i should first normalize the label to [0,1],then used for training?  After test ,when i want to see the true value,then i should deNormalize the value to [0,1000]?

**Comment by keunwoochoi** [comment]
Yes to all the questions. :)
